proposals/lip-0052.md

```diff
@@ -94,50 +95,98 @@ This function is used to transfer ownership of NFTs across chains in the Lisk ec
 
 #### recover
 
-This function should only be called by the Interoperability module to trigger the recovery of NFTs escrowed to terminated chains.
+This function should only be called by the [Interoperability module][lip-0045] to trigger the recovery of NFTs escrowed to terminated chains.
 
 ## Specification
 
-### Constants and Notations
+### Notation and Constants
 
 The following constants are used throughout the document:
 
-| Name                                  | Type    | Value           |
-|---------------------------------------|---------|-----------------|
-| **Interoperability Constants**        |         |                 |
-| `MIN_RETURN_FEE`                      | uint64  | 1000            |
-| **NFT Module Constants**              |         |                 |
-| `MODULE_ID_NFT`                       | uint32  | TBD             |
-| `COMMAND_ID_TRANSFER`                 | uint32  | 0               |
-| `COMMAND_ID_CROSS_CHAIN_TRANSFER`     | uint32  | 1               |
-| `CROSS_CHAIN_COMMAND_ID_TRANSFER`     | uint32  | 0               |
-| `CHAIN_ID_ALIAS_NATIVE`               | uint32  | 0               |
-| `NFT_NOT_LOCKED`                      | uint32  | `MODULE_ID_NFT` |
-| `MAX_BYTE_LENGTH_ATTRIBUTES`          | uint32  | 9 * 1024        |
-| `CCM_STATUS_OK`                       | uint32  | 0               |
-| `CCM_STATUS_NFT_NOT_SUPPORTED`        | uint32  | 64              |
-| `CCM_STATUS_NFT_PROTOCOL_VIOLATION`   | uint32  | 65              |
-| **Store Constants**                   |         |                 |
-| `STORE_PREFIX_NFT`                    | bytes   | 0x00 00         |
-| `STORE_PREFIX_USER`                   | bytes   | 0x80 00         |
-| `STORE_PREFIX_COLLECTION`             | bytes   | 0xc0 00         |
-| `STORE_PREFIX_AVAILABLE_COLLECTION`   | bytes   | 0xd0 00         |
-| `STORE_KEY_LENGTH_NFT`                | uint32  | 16              |
-| **General Constants**                 |         |                 |
-| `EMPTY_BYTES`                         | bytes   | ""              |
-| `ADDRESS_LENGTH`                      | uint32  | 20              |
+| Name                                   | Type   | Value                |
+| -------------------------------------- | ------ | -------------------- |
+| **Interoperability Constants**         |        |                      |
+| `MIN_RETURN_FEE`                       | uint64 | 1000                 |
+| **NFT Module Constants**               |        |                      |
+| `MODULE_NAME_NFT`                      | string | "nft"                |
+| `COMMAND_NAME_TRANSFER`                | string | "transfer"           |
+| `COMMAND_NAME_CROSS_CHAIN_TRANSFER`    | string | "transferCrossChain" |
+| `CROSS_CHAIN_COMMAND_NAME_TRANSFER`    | string | TBD                  |
+| `CHAIN_ID_ALIAS_NATIVE`                | bytes  | 0x 00 00 00 00       |
```

Comment:
I leave here the general comment about the 'chain ID alias' concept. I think we should follow the same approach we took for the Token module and remove this concept from the protocol. NFT would then just be identified by the ID of the chain also on the native chain. The reason why we introduced this alias was that before a chain not yet connected to the network did not have a chain ID at all. With the new changes, there is already a concept of chain ID independent from Interoperability (see https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#chain-identifiers). 

In practice, this would simply mean that we remove `CHAIN_ID_ALIAS_NATIVE` from the LIP (taking care that everything works out)

Reply:
Yes, I was initially confused about the chain ID alias, and will be glad to remove it now that it's not necessary anymore.